### PR TITLE
Adding new sensor placeholders

### PR DESCRIPTION
**Pull request by metonimie**
You can test the server with

python3 manage.py runserver

in the cserver directory.

### CODE REVIEW

File: BLECarPiZeroW/ccserver/sensors/urls.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from django.conf.urls import url
-from sensors.views import DemoSensor
+from sensors.views import DemoSensor, UltrasonicSensor, SpeedSensor
 
 urlpatterns = [
+    url(r'^speed/$', SpeedSensor.as_view(), name="sensors-soeed"),
```
Comment (vnemes) [inline BLECarPiZeroW/ccserver/sensors/urls.py]: typo "sensors-soeed" -> "sensors-speed"

**Comment by metonimie** [comment]
Fixed, also new module

**Comment by vnemes** [comment]
Looks good.
Pull request to be merged.

**Comment by metonimie** [comment]
done

**Pull request merged**